storage/spring-cloud-azure-starter-storage-file-share/storage-file-sample/README.md

```diff
@@ -0,0 +1,187 @@
+
+#### Run with Bash
+
+```shell
+# In the root directory of the sample
+# Initialize your Terraform configuration
+terraform -chdir=./terraform init
+
+# Apply your Terraform Configuration
+terraform -chdir=./terraform apply -auto-approve
+
+```
+
+#### Run with Powershell
+
+```shell
+# In the root directory of the sample
+# Initialize your Terraform configuration
+terraform -chdir=terraform init
+
+# Apply your Terraform Configuration
+terraform -chdir=terraform apply -auto-approve
+
+```
+
+It may take a few minutes to run the script. After successful running, you will see prompt information like below:
+
+```shell
+
+azurerm_resource_group.main: Creating...
+azurerm_resource_group.main: Creation complete after 5s ...
+azurerm_storage_account.application: Creating...
+azurerm_storage_account.application: Still creating... [10s elapsed]
+azurerm_storage_account.application: Creation complete after 36s ...
+azurerm_storage_share.application: Creating...
+azurerm_storage_share.application: Creation complete after 3s ...
+
+...
+Apply complete! Resources: 5 added, 0 changed, 0 destroyed.
+
+```
+
+You can go to [Azure portal](https://ms.portal.azure.com/) in your web browser to check the resources you created.
+
+### Export Output to Your Local Environment
+Running the command below to export environment values:
+
+#### Run with Bash
+
+```shell
+source ./terraform/setup_env.sh
+```
+
+#### Run with Powershell
+
+```shell
+terraform\setup_env.ps1
+```
+
+If you want to run the sample in debug mode, you can save the output value.
+
+```shell
+AZURE_STORAGE_ACCOUNT=...
+STORAGE_SHARE_NAME=...
+STORAGE_ACCOUNT_KEY=...
+```
+
+## Run Locally
+
+### Run the sample with Maven
+
+In your terminal, run `mvn clean spring-boot:run`.
+
+```shell
+mvn clean spring-boot:run
+```
+
+### Run the sample in IDEs
+
+You can debug your sample by adding the saved output values to the tool's environment variables or the sample's `application.yaml` file.
+
+* If your tool is `IDEA`, please refer to [Debug your first Java application](https://www.jetbrains.com/help/idea/debugging-your-first-java-application.html) and [add environment variables](https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html#add-environment-variables).
+
+* If your tool is `ECLIPSE`, please refer to [Debugging the Eclipse IDE for Java Developers](https://www.eclipse.org/community/eclipse_newsletter/2017/june/article1.php) and [Eclipse Environment Variable Setup](https://examples.javacodegeeks.com/desktop-java/ide/eclipse/eclipse-environment-variable-setup-example/).
+
+## Verify This Sample
+1. Write and read a file.  
+    1.1 Send a POST request to update file contents.
+    ```shell
+    curl http://localhost:8080/file/file1.txt -d "new message" -H "Content-Type: text/plain"
+    ```
+    1.2 Verify by sending a GET request.  
+    ```shell
+    curl -XGET http://localhost:8080/file/file1.txt
+    ```
```

Comment:
I got warnnings like below
<img width="656" alt="image" src="https://user-images.githubusercontent.com/4465723/179649654-afc987e4-e2db-4b6b-8b34-86c29ab5f7d4.png">

How about change `curl -XGET http://localhost:8080/file/file1.txt` to 
`curl -XGET http://localhost:8080/file/file1.txt --output file1.txt`

Reply:
We will try to see how to set the `content-type` to the file resource to solve this warning.